src/util/initEvents.ts

```diff
@@ -113,26 +113,35 @@ const scrollAtEdge = (() => {
   return { start, stop }
 })()
 
+let lastPath: Path | null
+
+/** Update the LastPath. */
+export const updateLastPath = (path: Path) => {
+  lastPath = path
+}
```

Comment:
A mutable global variable is considered a bad practice and should be avoided. A mutable variable with a limited scope (under closure) can occasionally be justified. For state that spans multiple files, we really should stick to proper state management solutions, like Redux or our internal ministore.